### DISCUSSION TITLE
Indexer registration issue

### DISCUSSION THREAD

**Discussion by ivankhrenkov777** [subquery/subql#998] [Frontier TestNet - Bug Reporting]
Registering indexer doesn't work - after clicking the button, metamask doesn't offer to approve the transaction
![image](https://user-images.githubusercontent.com/94184412/167415442-50258300-a981-4d46-906a-02e176617fc1.png)

**Comment by mzxyz** [comment]
Can I ask was there any errors when press the button?

**Reply by ivankhrenkov777** [reply]
No errors occurred, just after pressing the button nothing happened. But It was fixed in  0.19.0.

**Reply by mzxyz** [reply]
Got it, it just due to the ipfs service down and the indexer metadata can not publish to ipfs, we will better handle the error.
